CommandDotNet.Tests/FeatureTests/Arguments/ModelValidation.cs

```diff
@@ -0,0 +1,118 @@
+using CommandDotNet.Attributes;
+using CommandDotNet.Models;
+using CommandDotNet.Tests.ScenarioFramework;
+using CommandDotNet.Tests.Utils;
+using FluentValidation;
+using FluentValidation.Attributes;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace CommandDotNet.Tests.FeatureTests.Arguments
+{
+    public class ModelValidation
+    {
+        private readonly ScenarioVerifier _verifier;
+
+        public ModelValidation(ITestOutputHelper output)
+        {
+            _verifier = new ScenarioVerifier(output);
+        }
+
+        [Fact]
+        public void Help_DoesNotIncludeValidation()
```

Comment:
@bilal-fazlani here's an example of running a scenario within a standard test method.  This approach works better with MS, R# & Rider test runners.  I may end up converting the others to this style.